shared/js/loadApp.js

```diff
@@ -56,7 +58,13 @@ module.exports = function (callback) {
   }
 
   if (window.dashboard && dashboard.project) {
-    promise = promise.then(dashboard.project.load);
+    promise = promise.then(dashboard.project.load)
+    .then(function () {
+      if (dashboard.project.hideBecauseAbusive()) {
+        renderAbusive();
+        return $.Deferred().reject();
```

Comment:
I'm not sure this does anything if no one is waiting for the `$.Deferred` to resolve.